src/materials/ConcreteASREigenstrain.C

```diff
@@ -23,65 +23,79 @@ ConcreteASREigenstrain::validParams()
   params.makeParamRequired<Real>("compressive_strength");
   params.makeParamRequired<Real>("tensile_strength");
 
-  params.addRequiredCoupledVar("temperature", "Coupled temperature");
+  params.addRequiredCoupledVar(
```

Comment:
Do addRequiredCoupledVar not require units?  You added units for addCoupledVar above,  the units were temperature.  I think it should be K since the latency_activation_energy is K.   I"m assuming K is temperature

Reply:
I'll add that.